crates/comments/src/directive.rs

```diff
@@ -50,6 +56,29 @@ fn parse_format_directive(text: &str) -> Option<Directive> {
     }
 }
 
+fn parse_special_directive(text: &str) -> Option<Directive> {
+    if let Some(directive) = parse_special_skip_file(text) {
+        return Some(directive);
+    }
+
+    None
+}
+
+// https://github.com/posit-dev/air/issues/219
+fn parse_special_skip_file(text: &str) -> Option<Directive> {
+    // Convention used in the R community (Roxygen, Rcpp, etc)
+    if text.starts_with("Generated by") {
+        return Some(Directive::Format(FormatDirective::SkipFile));
+    }
+
+    // Convention used in some programming language communities
+    if text.contains("@generated") {
+        return Some(Directive::Format(FormatDirective::SkipFile));
+    }
```

Comment:
I would vote against doing this unless the community actively requested this from us, I would like to keep this feature as tightly scoped as possible